src/oscar/templates/oscar/dashboard/catalogue/product_update.html

```diff
@@ -172,6 +172,9 @@ <h3>{% trans "Upload, change or remove images" %}</h3>
                                                 {% include "dashboard/partials/product_images.html" with form=image_form %}
                                             {% endfor %}
                                         </ol>
+                                        <p class="help-block">
```

Comment:
I don't think this help text is necessary. The user will realise as soon as they add one image that the formset grows automatically.

Reply:
> I don't think this help text is necessary. The user will realise as soon as they add one image that the formset grows automatically.

Can't agree with that. Behavior of the formset will change and help text should help to avoid user confusion.

In short, I prefer to leave explicit instructions and also not sure how would user make conclusion about image re-order.

Reply:
OK - in that case please translate the string and change grammar to "Drag images to re-order them. Space for additional images will appear when images are added."